### DISCUSSION TITLE
Does it support E80-900M2213S module ？

### DISCUSSION THREAD

**Discussion by norulers** [ExpressLRS/ExpressLRS#2945] [General]
Can I use 2 E80-900M2213S to DIY GemX? 
I noticed that an RF switch needs to be configured, and I'm not sure if ELRS supports it .

**Comment by pkendall64** [comment]
Yes ELRS does support configuring the RF switch parameters for the LR1121.
See https://github.com/ExpressLRS/ExpressLRS/pull/2824

**Comment by norulers** [comment]
Is there a configuration example available ？
![20240923083255](https://github.com/user-attachments/assets/7a6a5bf0-f482-4963-a9db-fe8ec7c16b5c)
This is an example configuration of ebyte, how to set parameters to make it work ？

**Reply by pkendall64** [reply]
From that it looks like you'd need `radio_rfsw_ctrl` to be `[7,0,2,3,1,0,4,0]`

**Reply by norulers** [reply]
Thank you for your reply.

**Reply by NanoNM** [reply]
Hi @norulers  Have you successfully made this tx？ 

i have a problem

ExpressLRS TX Module Booted...
Config version 7
Hal Init
LR1121 Reset
version is 
SX12XX_Radio_1 1
Read LR1121 #1 Use Case (0x03 = LR1121): 223
LR1121 #1 failed to be detected.
Begin Webupdater
Changing to AP mode
HTTPUpdateServer ready! Open http://elrs_tx.local in your browser

**Reply by norulers** [reply]
If using the same module, it is recommended to use external 3.3 to power the crystal oscillator

**Reply by NanoNM** [reply]
i fix it but i can not band 2.4 or 915 rx
